src/components/time-picker/index.js

```diff
@@ -2,16 +2,8 @@ import React from 'react';
 import Picker from './picker';
 import './index.scss';
 
-const getMinutes = () => {
-  let i = 0,
-    limit = 60,
-    minutes = [];
-  for (i; i < limit; i += 1) {
-    minutes.push(i < 10 ? '0' + i : '' + i);
-  }
-  return minutes;
-};
 
+const minutes = [...Array(60).keys()].map(_ => _.toString().padStart(2, "0"));
```

Comment:
can you please make **_** to some appropriate variable name like **minute**. proper naming makes it more understandable, thought this code is pretty small and self explanatory, still following the same convention everywhere keeps things uniform.